docs/bpmn-support.adoc

```diff
@@ -135,20 +136,34 @@ The default rendering uses `white` as fill color and `black` as stroke color.
 |The stroke & icon width may be adjusted
 |===
 
+
 [cols="1,1,4", options="header"]
-.Boundary Events
```

Comment:
I created only one part for Boundary, because the compensation boundary event is not interrupting & not non-interrupting.